src/components/DraggableList/useDraggableInPortal.ts

```diff
@@ -0,0 +1,43 @@
+import type {DraggableChildrenFn} from 'react-beautiful-dnd';
+import {useEffect, useRef} from 'react';
+import {createPortal} from 'react-dom';
+
+type DraggableInPortalProps = {
```

Comment:
I think we should rename this type so that it doesn't end with `Props`. This is a type for the argument to a custom hook that happens to return a render function. `useDraggableInPortal` is not a component, so I don't think appending `Props` to the type name isn't necessary 👍

Reply:
change was made